.github/workflows/release.yaml

```diff
@@ -20,6 +23,20 @@ jobs:
       BIN_NAME: otdfctl
     steps:
       - uses: actions/checkout@v4
+      - name: Verify user is authorized to release
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```

Comment:
```suggestion
          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
Isn't this the token or do we need something more?

Reply:
removed